public/modules/dashboards/views/fbf.client.view.html

```diff
@@ -800,4 +829,84 @@ <h2 class="h5 modal-title">{{ "warning" | translate }}</h2>
     </div>
   </div>
   <!-- end: IE message -->
+
+  <!-- start: Mailchimp modal -->
+  <div id="MailChimpModal" tabindex="-1" class="modal" role="dialog">
+    <div class="modal-dialog">
+      <!-- Modal content-->
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal">
+            <i class="fa fa-times-circle-o" aria-hidden="true"></i>
+          </button>
+          <h2 class="h5 modal-title">
+            {{ "Sign up to mailing list" | translate }}
+          </h2>
+        </div>
+        <div class="modal-body">
+          <link
+            href="//cdn-images.mailchimp.com/embedcode/slim-10_7.css"
+            rel="stylesheet"
+            type="text/css"
+          />
+          <style type="text/css">
+            #mc_embed_signup {
+              background: #fff;
+              clear: left;
+              font: 14px Helvetica, Arial, sans-serif;
+            }
+            /* Add your own Mailchimp form style overrides in your site stylesheet or in this style block.
+              We recommend moving this block and the preceding CSS link to the HEAD of your HTML file. */
+          </style>
+          <div id="mc_embed_signup">
+            <form
+              action="https://global.us20.list-manage.com/subscribe/post?u=acdf2ad027e3073babce9067d&amp;id=f3a585f6ba"
+              method="post"
+              id="mc-embedded-subscribe-form"
+              name="mc-embedded-subscribe-form"
+              class="validate"
+              target="_blank"
+              novalidate
+            >
+              <div id="mc_embed_signup_scroll">
+                <label for="mce-EMAIL">Subscribe</label>
+                <input
+                  type="email"
+                  value=""
+                  name="EMAIL"
+                  class="email"
+                  id="mce-EMAIL"
+                  placeholder="email address"
+                  required
+                />
+                <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+                <div
+                  style="position: absolute; left: -5000px;"
+                  aria-hidden="true"
+                >
+                  <input
+                    type="text"
+                    name="b_acdf2ad027e3073babce9067d_f3a585f6ba"
```

Comment:
This string also shouldn't be hard-coded, as it contains a specific value to 'our' account.